utilities/blob_db/blob_db.h

```diff
@@ -71,7 +71,12 @@ struct BlobDBOptions {
   // what compression to use for Blob's
   CompressionType compression = kNoCompression;
 
-  // Disable all background job.
+  // If enabled, blob DB periodically cleanup stale data by rewriting remaining
+  // live data in blob files to new files. By disabling garbage collection,
+  // blob files will be cleanup based on TTL.
+  bool disable_auto_garbage_collection = true;
```

Comment:
What's the reason for disabling auto gc? auto gc should be enabled by default in my opinion.

Reply:
Oh never mind. Just saw that you already mentioned it in the summary that  you want to disable it until fully tested. (Sorry, should have clearly read the summary first). 
Sure, works for me.

Reply:
I personally dislike negative names in options, as it make it a little difficult for someone to wrap their head around when you set true or false. For example, if `disable_auto_garbage_collection = false`, it is like you have to think about double negation in your head to understand that garbage collection is actually enabled. :)
So, I'd prefer if the option is changed to, say, just `garbage_collection` (or `enable_garbage_collection`). May be we don't need even an 'auto' in the option name.